client/app/components/QueryEditor.jsx

```diff
@@ -1,347 +0,0 @@
-    // Lineup only mac
-    editor.commands.bindKey({ win: null, mac: "Ctrl+P" }, "golineup");
-    editor.commands.bindKey({ win: "Ctrl+Shift+F", mac: "Cmd+Shift+F" }, this.formatQuery);
-
-    // Reset Completer in case dot is pressed
-    editor.commands.on("afterExec", e => {
-      if (e.command.name === "insertstring" && e.args === "." && editor.completer) {
-        editor.completer.showPopup(editor);
-      }
-    });
-
-    QuerySnippet.query(snippets => {
-      const snippetManager = snippetsModule.snippetManager;
-      const m = {
-        snippetText: "",
-      };
-      m.snippets = snippetManager.parseSnippetFile(m.snippetText);
-      snippets.forEach(snippet => {
-        m.snippets.push(snippet.getSnippet());
-      });
-      snippetManager.register(m.snippets || [], m.scope);
-    });
-
-    editor.focus();
-    this.props.listenForResize(() => editor.resize());
-    this.props.listenForEditorCommand((e, command, ...args) => {
-      switch (command) {
-        case "focus": {
-          editor.focus();
-          break;
-        }
-        case "paste": {
-          const [text] = args;
-          editor.session.doc.replace(editor.selection.getRange(), text);
-          const range = editor.selection.getRange();
-          this.props.updateQuery(editor.session.getValue());
-          editor.selection.setRange(range);
-          break;
-        }
-        default:
-          break;
-      }
-    });
-  };
-
-  updateSelectedQuery = selection => {
-    const { editor } = this.refEditor.current;
-    const doc = editor.getSession().doc;
-    const rawSelectedQueryText = doc.getTextRange(selection.getRange());
-    const selectedQueryText = rawSelectedQueryText.length > 1 ? rawSelectedQueryText : null;
-    this.setState({ selectedQueryText });
-    this.props.updateSelectedQuery(selectedQueryText);
-  };
-
-  updateQuery = queryText => {
-    this.props.updateQuery(queryText);
-    this.setState({ queryText });
-  };
-
-  formatQuery = () => {
-    Query.format(this.props.dataSource.syntax || "sql", this.props.queryText)
-      .then(this.updateQuery)
-      .catch(error => notification.error(error));
-  };
-
-  toggleAutocomplete = state => {
-    this.setState({ autocompleteQuery: state });
-    localOptions.set("liveAutocomplete", state);
-  };
-
-  componentDidUpdate = () => {
-    // ANGULAR_REMOVE_ME  Work-around for a resizing issue, see https://github.com/getredash/redash/issues/3353
-    const { editor } = this.refEditor.current;
-    editor.resize();
-  };
-
-  render() {
-    const modKey = KeyboardShortcuts.modKey;
-
-    const isExecuteDisabled = this.props.queryExecuting || !this.props.canExecuteQuery;
-
-    return (
-      <section style={{ height: "100%" }} data-test="QueryEditor">
-        <div className="container p-15 m-b-10" style={{ height: "100%" }}>
-          <div
-            data-executing={this.props.queryExecuting}
-            style={{ height: "calc(100% - 40px)", marginBottom: "0px" }}
```

Comment:
@kravets-levko it seems this `height: calc(100% - 40px)` was important in older versions to avoid issues:
![chrome-67-calc](https://user-images.githubusercontent.com/3356951/71220542-a6b53e00-22a7-11ea-8547-443c0f97072e.gif)

Reply:
I installed Chromim@65 and reproduced but there. So fix will be available soon (I hope without `calc` :grin:)

Reply:
@gabrieldutra This is the actual fix, everything else is just refactor (I collected Ace styles to a single place). Now the only diffecrence Percy shows are buttons :tada: 
https://github.com/getredash/redash/blob/fb50a269da49bf13eb775a4e111aa0d3735b5ffa/client/app/components/queries/QueryEditor/index.less#L14-L21